extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -192,6 +192,9 @@ class ReplicationStatusProcessor {
                 return this.taskScheduler.push({ task, entry: sourceEntry },
                     sourceEntry.getCanonicalKey(), done);
             });
+        } else if (task) {
```

Comment:
The `else` is not necessary (no big deal but I have memories of the linter complaining about this).

Reply:
I vote for removing unnecessary `else` statements as well.

Reply:
Sounds good to me. 😄